### ISSUE TITLE
Ch-3 won't knit

### ISSUE THREAD

**Issue by liza-wood** [ryanpeek/aggiedown#11]
When knitting the template after Tok correction issue #6 , I get: 
```
! Missing number, treated as zero.
<to be read again> 
                   (
l.574 \begin
            {minipage}[b]{\linewidth}\centering 
```
System information is:

```
> Sys.info()
                                                                                           sysname 
                                                                                          "Darwin" 
                                                                                           release 
                                                                                          "22.6.0" 
                                                                                           version 
"Darwin Kernel Version 22.6.0: Wed Jul  5 22:21:56 PDT 2023; root:xnu-8796.141.3~6/RELEASE_X86_64" 
```

**Comment by liza-wood** [comment]
I have tried removing various elements of Chapter 3 to see what is bugging and it is the direct/manual table in line 33. When I remove I get a new error:

```
Quitting from lines 167-168 [uwlogo] (thesis.Rmd)
                                                                                                               
Error in `include_graphics()`:
! could not find function "include_graphics"
Execution halted
```

**Comment by ryanpeek** [comment]
This should be addressed now with #14
